gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -924,6 +929,36 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
       ret = filter->loaded = TRUE;
       break;
     }
+    case GTT_CLAMP:
+    {
+      gchar **strv = NULL;
+
+      if (!g_regex_match_simple (REGEX_CLAMP_OPTION, filter->option, 0, 0)) {
+        ml_loge
+            ("%s: clamp: \'%s\' is not valid option string: it should be in the form of [CLAMP_MIN:CLAMP_MAX]\n",
+            filter_name, filter->option);
+        break;
+      }
+
+      strv = g_strsplit (filter->option, ":", 2);
+
+      filter->data_clamp.min = g_ascii_strtod (strv[0], NULL);
+      if (errno == ERANGE) {
+        ml_loge ("%s: clamp: CLAMP_MIN value has invalid range\n", filter_name);
+        g_strfreev (strv);
+        break;
+      }
+      filter->data_clamp.max = g_ascii_strtod (strv[1], NULL);
```

Comment:
ensure that data_clamp.max > data_clamp.min?

Reply:
Yes, updated.